docs/tutorials/setting-up-ethernet-over-usb-on-linux-command-line.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Setting Up Ethernet Over USB on Linux with Command Line
```

Comment:
What tools are you using? The name of the tools should be added to the title. For example, it is possible to configure NetworkManager from the command line. If I was doing this, I would make the title something like "Setting Up Ethernet Over USB on Linux using NetworkManager via the Command Line"